test/utils/image/manifest.go

```diff
@@ -48,7 +48,7 @@ func (i *ImageConfig) SetVersion(version string) {
 }
 
 var (
-	AdmissionWebhook         = ImageConfig{e2eRegistry, "k8s-sample-admission-webhook", "1.10v2", true}
+	AdmissionWebhook         = ImageConfig{e2eRegistry, "k8s-sample-admission-webhook", "1.10v3", true}
```

Comment:
Seems like we may have the inverse backward compatability problem from aggregator. Should look at having a range of versions we run the test against.

Reply:
Hmm that's true. Although I don't think there's anything about the k8s-sample-admission-webhook that is k8s version dependent

Reply:
should probably bump this to be 1.12 at least, since that's where this change is going

Reply:
I agree that the version should reflect the version was build against. That having been said we should probably run the test with a webhook built against every supported version of apimachinery/api/apiextensions-apiserver/client-go. The point of the test is to catch if a code change breaks a supported legacy webhook.